packages/storybook/README.md

```diff
@@ -0,0 +1,72 @@
+# @builder.io/storybook
+
+Addon for integrating [Builder.io](https://www.builder.io) to allow drag and drop page building inside storybook. 
```

Comment:
We should host the storybook build somewhere so we can link people to be able to just click to try it out